### ISSUE TITLE
Deployment on ICM cluster fails due to missing realpath command

### ISSUE THREAD

**Issue by grzanka** [garstka/idact#6]
I've tried to deploy jupyter notebook on ICM cluster (topola) using idact, but the command `nodes[0].deploy_notebook()` fails due to missing `realpath` command:

```
RuntimeError: run() received nonzero return code 127 while executing!

Requested: realpath ~/.idact/runtime/IUrQpj3ZKggDReHS4NEiZQ8BRQY4rYIA
Executed: /bin/bash --noprofile -l -c "realpath ~/.idact/runtime/IUrQpj3ZKggDReHS4NEiZQ8BRQY4rYIA"

NoneType: None
```

Testing it manually:
```
[plgkongruencj@t10-3 ~]$ /bin/bash --noprofile -l -c "realpath ~/.idact/runtime/IUrQpj3ZKggDReHS4NEiZQ8BRQY4rYIA"
/bin/bash: realpath: command not found
```

Cluster details:
```
{'icm': Cluster(login.icm.edu.pl, 22, plgkongruencj, auth=AuthMethod.PUBLIC_KEY, key='/home/grzanka/.ssh/id_rsa', install_key=False, disable_sshd=False),
cluster.config.setup_actions.jupyter = ['plgrid/tools/python/3.5.2']
cluster.config.use_jupyter_lab = True
```

Idact is installed on cluster worker nodes.

Note that python, jupyter and other whole module system is not available on access node of TOPOLA ICM cluster, necessary software is accessible only from worker nodes.

**Comment by grzanka** [comment]
A workaround:
```
wget https://raw.githubusercontent.com/morgant/realpath/master/realpath
chmod 755 realpath
mkdir -p $HOME/bin
mv realpath $HOME/bin
```

**Comment by garstka** [comment]
Fixed in 63e59e9075.
Deployed 0.5.1: 917d016.

Now using readlink from coreutils instead of realpath.
